test_changes.gemspec

```diff
@@ -21,8 +21,8 @@ Gem::Specification.new do |spec|
     f.match(%r{^(test|spec|features)/})
   end
 
-  spec.bindir        = "exe"
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.bindir        = "bin"
+  spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
```

Comment:
bundler's [recommending a new convention](http://bundler.io/blog/2015/03/20/moving-bins-to-exe.html) to put the executables in an exe directory. However, I'm okay with moving the executable back to bin since the scripts there will be removed in this pull request.